lib/types.js

```diff
@@ -62,6 +63,7 @@ module.exports = function (modelTypes) {
   modelTypes.registerType(Array);
   modelTypes.registerType(GeoPoint);
   modelTypes.registerType(Object);
+  modelTypes.registerType(ObjectId);
```

Comment:
Is there any way how to register `ObjectID` as an alias for `ObjectId`? Or is it automatically handled?